pkg/cli/verifier/envoy_config.go

```diff
@@ -147,15 +147,13 @@ func (v *EnvoyConfigVerifier) verifySource() Result {
 
 	// Retrieve route configs
 	var routeConfigs []*xds_route.RouteConfiguration
-	if v.configAttr.trafficAttr.AppProtocol == constants.ProtocolHTTP {
-		configs := config.Routes.GetDynamicRouteConfigs()
-		for _, r := range configs {
-			routeConfig := &xds_route.RouteConfiguration{}
-			//nolint: errcheck
-			//#nosec G104: Errors unhandled
-			r.GetRouteConfig().UnmarshalTo(routeConfig)
-			routeConfigs = append(routeConfigs, routeConfig)
-		}
+	configs := config.Routes.GetDynamicRouteConfigs()
+	for _, r := range configs {
+		routeConfig := &xds_route.RouteConfiguration{}
+		//nolint: errcheck
+		//#nosec G104: Errors unhandled
+		r.GetRouteConfig().UnmarshalTo(routeConfig)
```

Comment:
Why not check the error here and return an unknown result?

Reply:
I couldn't find a scenario where the unmarshaling fails. Wanted to avoid unnecessary checks around this if we can't test it.